decidim-core/app/views/decidim/pages/_tabbed.html.erb

```diff
@@ -1,45 +1,38 @@
-<div class="wrapper">
-  <div class="row column">
-    <div class="page-title-wrapper">
-      <h1 class="heading1 page-title">
+<%# NOTE: this page does not use a layout %>
```

Comment:
Is this page supposed to work on mobile yet? It doesn't seem to work for me as this is what I get:

![Mobile tabbed page](https://user-images.githubusercontent.com/864340/183088240-956e047f-32bf-40f9-9633-7e7bf9c23fc2.png)